README.md

```diff
@@ -51,12 +52,13 @@ The project may work with other versions of these tools but they are not tested.
 
 ### Installation
 
-Clone the repository, its submodule, and run the container
+Clone the repository, its submodule, and run the container:
+
 ```bash
 git clone git@github.com:gnosis/dex-services.git
 cd dex-services
 git submodule update --init
-cd dex-contracts 
+cd dex-contracts
 npm ci && npx truffle compile
```

Comment:
This should probably be updated to use `yarn` or `yarn install` (they should be equivalent) instead of `npm ci`.

Reply:
```suggestion
yarn install && npx truffle compile
```